demo/bad_packages.md

```diff
@@ -49,111 +79,175 @@ vary):
 To start up the GUAC visualizer, follow
 [this guide](https://github.com/guacsec/guac-visualizer/tree/main/docs/setup.md).
 
-## New security incident!
+## New security incident
 
-A new security incident has occurred, and there's no CVE or report for it yet,
-but listening on various communities, we know that a particular package is
-affected, more specifically the package "golang/github.com/dougm/pretty" has
-been found to have a vulnerability.
+A new security incident has occurred and listening on various communities, we
+know that a particular package is affected, more specifically the package
+"maven/org.apache.logging.log4j/log4j-core" has been found to have a
+vulnerability (yikes!).
 
-The first step we can take is to mark this packages as bad. We can do this by
+The first step we can take is to mark this package as bad. We can do this by
 using the `guacone certify` command with the `-good=false` flag which asserts
 that it is a negative certification (instead of a positive one), as well as a
-`justification` flag to indicate why it is bad. In this cause it is a new
-undisclosed vulnerability.
+`justification` flag to indicate why it is bad. In this case, it is a critical
+vulnerability.
+
+```bash
+bin/guacone certify --good=false --justification="never use this version of log4j" --type package "pkg:maven/org.apache.logging.log4j/log4j-core@2.8.1"
+```
+
+an output will contain the following meaning that we have successfully added
+"CertifyBad":
 
 ```bash
-bin/guacone certify --good=false -n --justification="pretty bad undisclosed vuln" --type package pkg:golang/github.com/dougm/pretty
```

Comment:
this package does not have dependents...See [deps.dev link](https://deps.dev/go/github.com%2Fdougm%2Fpretty/v0.0.0-20160325215624-add1dbc86daf/dependents). Might have been a logic bug in the UI code returning false information? Removed the package for a simpler use case